### DISCUSSION TITLE
Consulta

### DISCUSSION THREAD

**Discussion by rauoliv** [Launch-X-Latam/MisionFrontEnd#510] [Team Posts]
Buenas tardes, alguien sabe como se puede ver las grabaciones de las sesiones?

**Comment by alexisgohe** [comment]
Espero te sirva, aquí aparecen todas las grabaciones de FrontEnd https://teams.microsoft.com/l/channel/19%3AWJNKiwxGQtPIrd14ayE6FvxZqYOEO2lBT092fLCDGk01%40thread.tacv2/tab%3A%3A26fa4a52-a4dd-4bd2-820c-a3a9b076c38d?groupId=8c234d62-8e08-415a-8242-509fa69c5c40&tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&allowXTenantAccess=false
